### PR TITLE
Add notebook for converting mobilenet from PyTorch to TensorFlow Lite

### PR DESCRIPTION
**Pull request by banoffee-pie**
- Tidies up previous notebooks

### CODE REVIEW

**Review by andrewstanfordjason — CHANGES_REQUESTED** [review]
please add a little text under the: Analyse Model section to let the user know what the analysis is. Other than that this is looking good.

**Comment by ghost** [comment]
The mobilenet notebook looks good. Thanks @banoffee-pie! Do we need to keep the pytorchtokeras notebook?

**Comment by banoffee-pie** [comment]
> The mobilenet notebook looks good. Thanks @banoffee-pie! Do we need to keep the pytorchtokeras notebook?

They're using different libraries for the onnx -> tensorflow stage; I think @andrewstanfordjason wanted to compare for a few different models to see which is more accurate / has fewer operations?

**Comment by ghost** [comment]
> > The mobilenet notebook looks good. Thanks @banoffee-pie! Do we need to keep the pytorchtokeras notebook?
> 
> They're using different libraries for the onnx -> tensorflow stage; I think @andrewstanfordjason wanted to compare for a few different models to see which is more accurate / has fewer operations?

Ah, okay, got it. Have to think about how to add this into the documentation page here, https://github.com/xmos/ai_tools/blob/develop/docs/rst/pytorch.rst . We don't want people to use the tedious/inaccurate option.

**Comment by banoffee-pie** [comment]
> Ah, okay, got it. Have to think about how to add this into the documentation page here, https://github.com/xmos/ai_tools/blob/develop/docs/rst/pytorch.rst . We don't want people to use the tedious/inaccurate option.

[PINTO0309/onnx2tf](https://github.com/PINTO0309/onnx2tf) worked without any changes, but I had to apply a slightly odd fix for the other library (since it couldn't handle Relu6; hence it pointing to my fork of the library atm). Might it be better to only leave the [PINTO0309/onnx2tf](https://github.com/PINTO0309/onnx2tf) one in the README until I add more models and confirm that the other library works for others too?

Or perhaps label the other library as a "backup option"?

**Comment by banoffee-pie** [comment]
Last commit on the other library was also 2 years ago; I'm not sure if it's being actively maintained.

**Review by andrewstanfordjason — APPROVED** [review]
